### ISSUE TITLE
0.9.0 issues 'expected... but found' warnings for shorthand and expanded properties

### ISSUE THREAD

**Issue by ghost** [CSSLint/csslint#219]
I updated CSSLint to 0.9.0 and started getting warnings like this:

```
styles.css
1: warning at line 2, col 2
Expected all of (<border-width>, <border-style>, <color>) but found 'none'.
 border: none;

styles.css
2: warning at line 3, col 2
Expected end of value but found 'inset'.
 box-shadow: 1px 20px 1px #5d6872 inset;

```

0.8.5 reports zero warnings for the same file.

Contrived sample file:

``` css
div {
 border: none;
 box-shadow: 1px 20px 1px #5d6872 inset;
}
```

The command I'm running:
$ rhino csslint-rhino.js styles.css
